.github/workflows/java-ci.yml

```diff
@@ -17,10 +17,11 @@ jobs:
         java: [8, 11, 17]
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Setup java
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
+          distribution: "zulu"
```

Comment:
https://github.com/actions/setup-java
> V2 requires you to specify distribution along with the version. V1 defaults to Azul Zulu OpenJDK

Just use the default distribution we have been using so far.

Reply:
side notes, seems setup-java v2 havent get into node update. V2 already the latest, so even we still have warning now it should be gone once setup-java update their node version